Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Utilities/RectTransformCubeScaler.cs

```diff
@@ -17,6 +17,10 @@ public class RectTransformCubeScaler : MonoBehaviour
         private RectTransform rectTransform;
         private Vector2 prevRectSize = Vector2.zero;
 
+        [SerializeField]
+        private Transform objectToScale = null;
```

Comment:
if this is serialized and visible in the editor we should document it (+tooltip)

Reply:
Good point.  I'll add a comment & tooltip here.

Reply:
Added a tooltip.  I think it explains the correct usage.